src/ParameterDefinition.php

```diff
@@ -187,19 +187,33 @@ private function resolveUnionType(ContainerInterface $container)
                     /** @psalm-suppress PossiblyNullReference */
```

Comment:
Why don't you add the non-null check, and remove @psalm-suppress ?

Reply:
This code don't changed in PR.